include/integration/simple_MC_integration.hpp

```diff
@@ -0,0 +1,283 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2021 Vissarion Fisikopoulos
+// Copyright (c) 2018-2021 Apostolos Chalkis
+
+// Contributed and/or modified by Suraj Choubey, as part of Google Summer of Code 2021 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+// Monte Carlo Integration algorithm used here : https://en.wikipedia.org/wiki/Monte_Carlo_integration#Overview
+
+#ifndef SIMPLE_MC_INTEGRATION_HPP
+#define SIMPLE_MC_INTEGRATION_HPP
+
+#include <iostream>
+#include <fstream>
+#include <random>
+#include <vector>
+#include "convex_bodies/hpolytope.h"
+#include "Eigen/Eigen"
+#include "generators/known_polytope_generators.h"
+#include "boost_random_number_generator.hpp"
+#include "cartesian_geom/cartesian_kernel.h"
+#include "random_walks/random_walks.hpp"
+#include "volume/volume_sequence_of_balls.hpp"
+#include "volume/volume_cooling_gaussians.hpp"
+#include "volume/volume_cooling_balls.hpp"
+#include "misc.h"
+
+typedef double NT;
+typedef Cartesian<NT> Kernel;
+typedef typename Kernel::Point Point;
+typedef std::vector<Point> Points;
+typedef HPolytope<Point> HPOLYTOPE;
```

Comment:
this is not a special type so calling it `Hpolytope` instead if capitilize, is OK

Reply:
changed `HPOLYTOPE` -> `Hpolytope`